server/athenian/api/internal/miners/github/bots.py

```diff
@@ -85,13 +93,43 @@ async def __call__(
                 ),
             ),
         )
-        bots = frozenset(
+        return frozenset(
             self._bots[0].union(
                 (r[0] for r in team_logins),
                 *(self._bots.get(mid, set()) for mid in meta_ids),
             ),
         )
-        return bots
+
+    async def get_account_bots(
+        self,
+        account: int,
+        meta_ids: tuple[int, ...],
+        mdb: Database,
+        sdb: Database,
+        cache: Optional[aiomcache.Client],
+    ) -> AccountBots:
+        """Return the account bots collection."""
+        all_bots = await self(account, meta_ids, mdb, sdb, cache)
+        global_bots = await self.extra(mdb)
+        return AccountBots(global_bots, all_bots - global_bots)
+
+
+@dataclasses.dataclass(frozen=True)
+class AccountBots:
+    """The collection of bots for an account.
+
+    Bots are partitioned into global (common to every account)
+    and local (specific of this account) groups
+
+    """
+
+    global_bots: frozenset[str]
```

Comment:
IMHO it is an improvement, not an alternative behavior. If we sometimes need the "account specific" bots separated from "extra" bots it's natural to have this API instead of doing:
```
bots = fetch_bots()
account_specific_bots = bots - fetch_bots.extra(mdb)
```

But of course I can easily delete  the new method.